filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1275,7 +1275,10 @@ void OpenGLDriver::createTimerQueryR(Handle<HwTimerQuery> tqh, int) {
 
 void OpenGLDriver::destroyVertexBuffer(Handle<HwVertexBuffer> vbh) {
     DEBUG_MARKER()
-    // Do nothing, because a VertexBuffer is just a collection of BufferObject handles.
+    if (vbh) {
```

Comment:
This is part of hunter-main. Do we need this in the PR?

Reply:
Removed. Thanks for updating hunter-main!